Dockerfile.dev

```diff
@@ -0,0 +1,23 @@
+# Dockerfile.dev is a stripped down version of Dockerfile that is used for development purposes.
+# It minimizes number of installs and builds to speed up the development process.
+# Run this on local host only.
+# Generated image is larger than production image. Do not use this for production.
+
+FROM --platform=${BUILDPLATFORM:-amd64} node:22 AS build_dev
+ARG COMMIT
+
+WORKDIR /usr/app
+
+RUN apt-get update && apt-get install -y g++ make python3 python3-setuptools
```

Comment:
do we really need to still install all of these even if we use a full node image, in that case we should consider slim image